src/desktop/space/element.rs

```diff
@@ -55,6 +72,11 @@ where
         damage: &[Rectangle<i32, Logical>],
         log: &slog::Logger,
     ) -> Result<(), R::Error>;
+
+    /// Returns they layer the elements schould be draw on, defaults to Top
+    fn layer(&self) -> RenderLayer {
```

Comment:
I am not sure about the name `layer`, given that it might be possible to confuse this with layer-shell, but I don't really have a better idea.

Reply:
Yeah I didn't come with a better name too, since it needs to line up with RenderElement. 
RenderElementLayer sounds to verbose to me, but could be one option?